wave_vision/tests/descriptor_tests/brisk_tests.cpp

```diff
@@ -164,9 +165,17 @@ TEST(BRISKTests, DISABLED_ComputeDescriptors) {
     FASTDetector fast;
     BRISKDescriptor brisk;
 
+    auto start = std::chrono::steady_clock::now();
+
     keypoints = fast.detectFeatures(image);
     descriptors = brisk.extractDescriptors(image, keypoints);
 
+    auto extract_time = std::chrono::duration_cast<std::chrono::milliseconds>(
```

Comment:
This was just to get an idea of how fast things are running, but the results don't really make sense tbh. Removing this.